doc/api/errors.md

```diff
@@ -450,18 +452,35 @@ added properties.
 
 #### error.code
 
-Returns a string representing the error code, which is always `E` followed by
-a sequence of capital letters, and may be referenced in `man 2 intro`.
+The `error.code` property is a string representing the error code, which is always
+`E` followed by a sequence of capital letters, and may be referenced in `man 2 intro`.
```

Comment:
I don't see them refed on linux in the syscall intro, I do see them in `man 3 errno`.